### PR TITLE
Extend pivot table funtionality

### PR DESCRIPTION
**Pull request by EugeneAndrosovPaser**
I am using your project and I have a need for advanced pivot table management.

Extensions and adaptations for our project are presented here. In order not to accompany and update both projects at once, I propose to make these changes to the master after the review and correction of comments.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,17 +1,12 @@
-module github.com/360EntSecGroup-Skylar/excelize/v2
+module github.com/EugeneAndrosovPaser/excelize/v2
```
Comment (xuri) [inline go.mod:1]: Please don't use private packages here.

File: go.sum [hunk 2]
```diff
@@ -1,3 +1,7 @@
+github.com/360EntSecGroup-Skylar/excelize/v2 v2.2.0 h1:5DuRTdH6M8yPjvFfBkACVmuk7SoTzmaB8yM6KVqEhP8=
```
Comment (xuri) [inline go.sum:1]: please kept original `go.mod` and `go.sum` files

**Review by xuri — CHANGES_REQUESTED** [review]
Thanks for your PR. I've left some comments. Please make sure to pass the CI.

**Comment by xuri** [comment]
LGTM, thank you @EugeneAndrosovPaser.

**Comment by EugeneAndrosovPaser** [comment]
Sorry for my absence. I had sick. Thank you for resolving my merge request.

**Pull request merged**